device/transport/mqtt/test/_mqtt_test.js

```diff
@@ -162,7 +162,7 @@ describe('Mqtt', function () {
       /*Tests_SRS_NODE_DEVICE_MQTT_18_045: [ The `sendOutputEvent` method shall connect the Mqtt connection if it is disconnected. ]*/
       it('connects the transport if currently disconnected', function (testCallback) {
         var transport = new Mqtt(fakeAuthenticationProvider, fakeMqttBase);
-        testConfig.sendFunc(transport, new Message('test'), function () {
+        testConfig.sendFunc(transport, new Message('test'), function (err, resp) {
```

Comment:
>err, resp [](start = 70, length = 9)

While this indeed is the function signature, you don't appear to be using either of these two arguments in the callback itself.